website/static/index.html

```diff
@@ -228,22 +212,29 @@ <h4 class="accordion-heading"><span><strong class="faq-q">What are the future pl
             <div class="icon-2 accordion-icon">keyboard_arrow_down</div>
           </div>
           <div class="accordion-item-content">
-            <p class="faq-a">Current contributors across Onehouse, Microsoft, Google, and others are planning to incubate OneTable into the Apache Software Foundation. Stay tuned for more updates.</p>
+            <p class="faq-a">Current contributors across Onehouse, Microsoft, Google, and others are planning to incubate XTable into the Apache Software Foundation. Stay tuned for more updates.</p>
```

Comment:
This is already done so it may be good to update this part if not now then in a follow-up.

Reply:
I updated language to indicate present tense of helping to incubate

Reply:
I actually re-read this and now deleted this section from the Q&A as it's not as relevant anymore